public-api/db-cdr/changelog-schema/db.changelog-63.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet id="db.changelog-64" author="srushtigangireddy" runAlways="true" onValidationFail="MARK_RAN">
+    <validCheckSum>ANY</validCheckSum>
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <columnExists tableName="survey_question_map" columnName="is_parent_question"></columnExists>
+        <columnExists tableName="survey_question_map" columnName="generate_counts"></columnExists>
+      </not>
+    </preConditions>
+    <addColumn tableName="survey_question_map">
+      <!-- Used to generate questioncounts for each survey -->
+      <column name="is_parent_question" type="int">
+        <constraints nullable="true"/>
+      </column>
+    </addColumn>
+    <addColumn tableName="survey_question_map">
+      <!-- Used to include / exclude questions in count generation -->
+      <column name="generate_counts" type="int">
+        <constraints nullable="true"/>
+      </column>
+    </addColumn>
+  </changeSet>
+
+</databaseChangeLog>
+
```

Comment:
is_parent_question is to display the counts of only parent questions on the tiles. Generate_counts is used as metadata and we do not generate count data of questions that have 0 for this.